tests/drivers/spi/spi_loopback/boards/frdm_mcxn236.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&flexcomm3_lpspi3 {
+	slow@0 {
```

Comment:
Could you document the pins that should be shorted together in this overlay file, similar to what is done for the I2C overlay?

Reply:
Updated.